pkg/controller/servicebinding/flag.go

```diff
@@ -0,0 +1,17 @@
+package servicebinding
+
+import "github.com/spf13/pflag"
+
+var (
+	sboFlagSet              *pflag.FlagSet
+	maxConcurrentReconciles int
+)
+
+func init() {
+	sboFlagSet = pflag.NewFlagSet("sbo", pflag.ExitOnError)
+	sboFlagSet.IntVar(&maxConcurrentReconciles, "max-concurrent-reconciles", 1, "max-concurrent-reconciles is the maximum number of concurrent Reconciles which can be run. Defaults to 1.")
```

Comment:
we should read the default value from an env var here, and assume 1 if the var is not found.

Reply:
> apart from having the commnad line arg, we should have ability to configure max workers via env var as well.

Sure, but this would mean change `flag` if it doesn't already support it, which we do not want to do in this work item; this leaves us with the option of introducing a broader support for communicating environment variables in another PR.

Reply:
Instead of fixing the value to `1`, we should try to read it from an env var. This PR should be about ability to configure max number of workers, not about additing just command line arg for that.

Reply:
In my perspective, if I had to decide what goes on, from a performance point of view it is much better to have the default value being the number of CPUs, since it will reflect any changes in the deployment automatically leaving the environment variable or command line switch as a resource for the developer when debugging or stressing the software.

Other than that, it seems to be trivial to extract the number of CPUs available: http://play.golang.org/p/cuaf2ZHLIx.

Reply:
Here there is an example on how to extend `pflag` with environment variables: https://scene-si.org/2020/04/28/extending-pflag-with-environment-variables/.

Reply:
Let's consider the number of workers being CPU cores + 1 the default, and extend pflag with environment variables in another changelist.